doc/DeveloperGuide.md

```diff
@@ -94,6 +94,12 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should be carry out operations within O(log N) time
+6. Should have an intuitive user interface
```

Comment:
Recall from tutorial that NFRs are constraints and therefore needs to be quantified, can this be quantified?

Reply:
So NFR 6 should be
6. Should have the relevant logos on buttons to tell what the button does.
Is that okay?